src/pages/Admin/Details/ValidationSchema.ts

```diff
@@ -126,7 +127,8 @@ const roomTitleSchema = Yup.string()
   .min(3, ({ min }) => `Name must be at least ${min} characters`);
 const roomUrlSchema = Yup.string()
   .required("Url is required!")
-  .min(3, ({ min }) => `Url must be at least ${min} characters`);
+  .min(3, ({ min }) => `Url must be at least ${min} characters`)
+  .test("url validation", "Enter correct url", isValidUrl);
```

Comment:
Maybe change "Enter correct url" to something like "Please enter a valid URL" or "Entered URL is invalid" or similar (i'm not 100% how this shows up in the app/etc.

Or maybe even something that explains the format the URL needs to be in, as that would be most helpful to the user?

---

Ref: https://github.com/jquense/yup#mixedtestname-string-message-string--function-test-function-schema

Reply:
Let's just keep it as `'Please enter a valid URL'` for now, we can worry about coming up with a better/more descriptive message in a future cleanup if we need to.